packages/curve-sampling/curve-sampling.0.2/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>" ]
+license: "GPL-3.0+"
+homepage: "https://github.com/Chris00/ocaml-curve-sampling"
+dev-repo: "git+https://github.com/Chris00/ocaml-curve-sampling.git"
+bug-reports: "https://github.com/Chris00/ocaml-curve-sampling/issues"
+doc: "https://Chris00.github.io/ocaml-curve-sampling/doc"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test & os = "linux"}
+]
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "gg"
+  "dune" {>= "1.3"}
+  "cppo" {build | with-test}
```

Comment:
```suggestion
  "cppo" {build}
```
The build flag should suffice here. Build dependencies are always included during tests